spec/features/user_registers_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'spec_helper'
+
+feature "user registers", {js: true, vcr: true} do
+  background do
+    visit register_path
+  end  
+  
+  scenario "with valid user info and valid card" do
+    fill_valid_user
+    fill_valid_card    
+    click_button "Sign Up"
+    expect(page).to have_content("Profile was successfully created. Please sign in.")
```

Comment:
You can look for that element:

```ruby
expect(page).to have_css('span.payment_errors')
```

I'm a bit less worried about text that comes from stripe though, as that will very rarely change, if ever.